modules/updating-clear.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-cluster-cli.adoc
+
+[id="updating-clear_{context}"]
+= Recovering when an update fails before it is applied
+
+If an update fails before it is applied, such as when the version that you
```

Comment:
How to define "before it is applied", this is very important for customer's understanding here.
QE try upgrade to a version that can not be found, oc client would throw error directly, no any history for this operation in clusterversion. `oc adm upgrade clear` would take no any effect.
QE also try upgrade to a version that is not signed, upgrade failed upon image validation with "the image may not be safe to use" error.  `oc adm upgrade clear` can not help me clear the error, would take no any effect either.
From the above scenario, I did not see what the command is used for.

Reply:
@smarterclayton, will you PTAL? I need to clarify when to run  `oc adm upgrade clear` to cancel a not-yet-started update. It's not working the way I expected.

Reply:
I'm removing this topic from the TOC until we can get feedback from Clayton about how to fix it.
I'm not deleting the file because I want to merge it and then fix it and add it to the TOC later.